desktop-widgets/divelistview.cpp

```diff
@@ -49,9 +49,6 @@ DiveListView::DiveListView(QWidget *parent) : QTreeView(parent),
 
 	resetModel();
 
-	// Update selection if all selected dives were hidden by filter
-	connect(&diveListNotifier, &DiveListNotifier::filterReset, this, &DiveListView::filterFinished);
-
```

Comment:
why is this removed?

Reply:
This is a gremlin. I have no idea how this became part of the PR. Should I submit a commit that undoes these three changes?

Reply:
I can take care of that when merging.